### ISSUE TITLE
Object detection. No cropping in evaluation config results in wrong validation

### ISSUE THREAD

**Issue by SergeyBykov1** [tensorflow/models#3068]
### System information
- **What is the top-level directory of the model you are using**: object_detection
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Linux Ubuntu 16.04
- **TensorFlow installed from (source or binary)**: binary for anaconda python
- **TensorFlow version (use command below)**: v1.4.0-rc1-11-g130a514 1.4.0
- **Bazel version (if compiling from source)**: N/A
- **CUDA/cuDNN version**: 6.0
- **GPU model and memory**: gtx1060 6gb
- **Exact command to reproduce**:

### Describe the problem
Describe the problem clearly here. Be sure to convey here why it's a bug in TensorFlow or a feature request.

I train the ssd mobilenet v1 model on the kitti dataset.
I want the model input to be 640x360 (aspect ratio 16:9 ~ 1.7778)
```
    image_resizer {
      fixed_shape_resizer {
        height: 360
        width: 640
      }
    }
```
Kitti dataset image resolution is 1242x375 (aspect ratio 3.312)

I use a following preprocessing in train_config to use subimage with standard aspect ratio.
```
  data_augmentation_options {
    random_crop_to_aspect_ratio {
      aspect_ratio: 1.7778
      overlap_thresh: 0.8
    }
  }
```

So the training images are first cropped to 667x375 and are then resized to 640x360, resulting in normal training picture.

But there is no such preprocessing during the evaluation step, which means that validation images 1242x375 are just resized to 640x360, which results in images with squeezed road objects.

Because of that I get poor accuracy on the validation accuracy chart in tensorboard, but the model actually performs well during the manual testing, when I feed it with 16:9 aspect ratio images.

Is there a way to reproduce the same cropping during the validation step?

**Comment by asimshankar** [comment]
@jch1 @tombstone @derekjchow @jesu9 @dreamdragon - mind taking a look?

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
